src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -22,6 +23,11 @@
 @Configuration
 @EnableConfigurationProperties
 @ConfigurationProperties(prefix = "datarepo")
+@ConditionalOnProperty(
+    prefix = "tdr",
+    name = "testNoDatabase",
```

Comment:
Having a boolean flag with a negative name usually makes it harder to understand how it's being used. Can this be done using a positive name instead?